.internal-ci/test/fs-integration/subaddress-receipt.py

```diff
@@ -0,0 +1,101 @@
+import asyncio
+import sys
+import subprocess
+import json
+import argparse
+repo_root_dir = subprocess.check_output("git rev-parse --show-toplevel", shell=True).decode("utf8").strip()
+sys.path.append("{}/python-library".format(repo_root_dir))
+
+from fullservice import FullServiceAPIv2 as v2
+from basic import wait_for_account_to_sync
+from FSDataObjects import Response, Account 
+
+
+async def main(config):
+    # General test setup
+    fs = v2()
+
+    while (await fs.get_wallet_status())['result']['wallet_status']['is_synced_all'] != True:
```

Comment:
In general, we'll want to be ok with KeyErrors that will arise if these keys in the [] notation don't exist, which means the test will likely crash since Im not sure how the error handling is setup. There's a mixture of `[]` and `get()` this file, but generally I prefer using `get()`.